website2/src/components/GoogleAnalytics.tsx

```
@@ -45,29 +44,22 @@ export function GoogleAnalytics({ measurementId }: GoogleAnalyticsProps) {
-        strategy="afterInteractive"
-        onError={(e) => {
-          console.error('Error loading Google Analytics:', e);
-        }}
       />
-      <Script
-        id="google-analytics"
-        strategy="afterInteractive"
-        onError={(e) => {
-          console.error('Error initializing Google Analytics:', e);
+      <script
+        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)